neosensory_python/__init__.py

```diff
@@ -158,12 +159,11 @@ async def vibrate_motors(self, motor_list):
                 16 ms period.
 
         """
-
-        motor_command_string = "motors vibrate \"{}\"\r\n".format(base64.
```

Comment:
`bytearray` probably works just fine here.  I think the encoding of the `"`'s may have been the real culprit.